src/pages/about-us.tsx

```diff
@@ -10,7 +11,7 @@ const AboutUs = () => (
       <Navbar />
     </header>
     <main>
-      <div>About Us</div>
+      <AboutUsIntroduction />
```

Comment:
Currently the `main` section `width` has no **max**. Please see `layout-home`.

Reply:
Already fixed it.